apprentice_teacher.rb

```diff
@@ -1,23 +1,17 @@
-class ApprenticeTeacher
+require_relative 'people'
+require_relative 'teacher'
+class ApprenticeTeacher < People
   attr_reader :age, :salary, :phase, :target_raise
   attr_accessor :name
```

Comment:
Most of these are common to both ApprenticeTeacher and SeniorTeacher -- how can you refactor?